charm_test.go

```diff
@@ -103,3 +102,57 @@ func (yh YamlHacker) Reader() io.Reader {
 	}
 	return bytes.NewBuffer(data)
 }
+
+func charmDirPath(c *gc.C, name string) string {
+	path := filepath.Join("internal/test-charm-repo/quantal", name)
```

Comment:
quantal is what we use everywhere for sample charms (including charmstore) - it was chosen because quantal is a very little used release, so appropriate as an indicator that it's test data, not real.